app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  validates :email, uniqueness: true, presence: true
```

Comment:
research need for presence:true

Reply:
Good call. Lets circle back once we have some time.